docs/_Docs_GetStarted.md

```diff
@@ -68,78 +73,79 @@ The final step for installation is copying over some of the files from our submo
 
-You can move the network forward by an hour with the following command.
+### Access with the Truffle console
 
+First, add a private key of your choice to the `truffle.js` configuration file:
 ```
-curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_increaseTime","params":[86400],"id": 1}' localhost:8545
+goerli: {
+      provider: () => {
+        return new HDWalletProvider("replace-with-private-key-when-using", "https://goerli.infura.io/v3/e21146aa267845a2b7b4da025178196d");
+      },
+      network_id: "5"
+    }
+  },
 ```
 
-Once you have moved the network forward by an hour, you can then mine a new block with the following command.
-
+Then, start up the truffle console and connect to testnet:
 ```
-curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_mine","params":[]}' localhost:8545
+$ yarn truffle console --network goerli
 ```
+In the truffle console, instantiate the IColonyNetwork interface for `glider-rc.1`:
+```
+truffle(goerli)> let IColonyNetwork = await IColonyNetwork.at("0x79073fc2117dD054FCEdaCad1E7018C9CbE3ec0B")
 
-Note that because reputation is awarded for the *previous* submission window, you will need to use the "fast-forward" command above to speed through at least 3 reputation updates before noticing a change in the miner's reputation.
-
-### Get Reputation from the Reputation Oracle
-
-The reputation mining client will answer queries for reputation scores locally over HTTP.
-
 ```
-http://127.0.0.1:3000/{reputationState}/{colonyAddress}/{skillId}/{userAddress}
+From here, you can create a new colony (with an ERC20 token already deployed):
+```
+truffle(goerli)> IColonyNetwork.createColony("your-erc20-token-address")
+```
+And find your colony's id (the newest created colony) after the transaction is mined:
 ```
 
-The oracle should be able to provide responses to any valid reputation score in all historical states, as well as the current state.
+truffle(goerli)> IColonyNetwork.getColonyCount()
+```
+### Local Development and Testing
 
-For example, you can get the reputation score of the miner in the current reputation state (which after three updates, will be `0x7ad8c25e960b823336fea83f761f5199d690c7b230e846eb29a359187943eb33`) using the address of the Meta Colony (`0x1133560dB4AebBebC712d4273C8e3137f58c3A65`), the skill tag of `2`, and the address of the miner (`0x3a965407ced5e62c5ad71de491ce7b23da5331a4`).
+You can start a local test node and deploy the contracts yourself using the locally installed `truffle` package.
 
 ```
-http://127.0.0.1:3000/0x7ad8c25e960b823336fea83f761f5199d690c7b230e846eb29a359187943eb33/0x1133560dB4AebBebC712d4273C8e3137f58c3A65/2/0x3a965407ced5e62c5ad71de491ce7b23da5331a4
+yarn run start:blockchain:client
+
+./node_modules/.bin/truffle migrate --reset --compile-all
 ```
 
-The oracle should return something similar to the following.
+To deploy all contracts and run all tests, run the following command.
```

Comment:
In #613 we have split the reputation mining tests away from the main test suite. After this is merged in addition to `test:contracts` we'll have `test:reputation` which will run reputation contract and miner client tests and checks their coverage.